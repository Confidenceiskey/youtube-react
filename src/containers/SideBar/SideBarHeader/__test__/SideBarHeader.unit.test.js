import React from 'react';
import { shallow } from 'enzyme';
import SideBarHeader from '../SideBarHeader';

describe('Various cases for SideBarHeader', () => {
  it('renders SideBarHeader with props.title=""', () => {
    const wrapper = (
      <SideBarHeader title='' />
    );
    expect(wrapper).toMatchSnapshot();
  });

  it('renders SideBarHeader with props.title="Heading"', () => {
    const wrapper = (
      <SideBarHeader title='Heading' />
    );
    expect(wrapper).toMatchSnapshot();
  });
});


