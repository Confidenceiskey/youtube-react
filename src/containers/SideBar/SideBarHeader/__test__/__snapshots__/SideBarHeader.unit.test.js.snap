// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Various cases for SideBarHeader renders SideBarHeader with props.title="" 1`] = `
<SideBarHeader
  title=""
/>
`;

exports[`Various cases for SideBarHeader renders SideBarHeader with props.title="Heading" 1`] = `
<SideBarHeader
  title="Heading"
/>
`;

exports[`renders SideBarHeader 1`] = `<SideBarHeader />`;
